src/create-reminder-signup/lambda/local.ts

```diff
@@ -10,9 +10,11 @@ function runLocal() {
 	process.env.Stage = 'DEV';
 
 	const event = {
+		path: '/create/one-off',
 		body: JSON.stringify({
 			email: 'test-reminders10@theguardian.com',
 			reminderPeriod: '2021-01-01',
+			reminderFrequencyMonths: 3,
```

Comment:
might be worth putting one-off and recurring example events into json files

Reply:
yeah that's a good idea - should we just have them there as examples? or do we want to actually use the contents of those files when running locally?

Reply:
Mainly as examples, though perhaps `local.ts` could load them? Not essential though

Reply:
I've pushed up some examples. For the moment I didn't load those in `local.ts` though - I was thinking for testing purposes it's actually slightly easier to work with the js objects than raw json

Reply:
👍